runtime/ops/spawn.rs

```diff
@@ -262,3 +270,16 @@ fn op_spawn_sync(
     },
   })
 }
+
+#[op]
+fn op_spawn_kill(
+  state: &mut OpState,
+  rid: ResourceId,
+  signal: String,
+) -> Result<(), AnyError> {
+  if let Ok(child_resource) = state.resource_table.get::<ChildResource>(rid) {
+    kill(child_resource.1 as i32, &signal)?;
+    return Ok(());
+  }
+  Err(type_error("Child process has already terminated."))
```

Comment:
```suggestion
  Err(type_error("Child process is already terminated."))
```

Is _is_ a better verb here?

"has terminated" isn't grammatical in English (unless the meaning is "has terminated itself").

Reply:
@nayeemrmn I noticed that you declined to accept this suggested change, but then later pushed a [commit](https://github.com/denoland/deno/pull/15339/commits/59738cad3520ed49afe8257a4b1c3379b11d97a4) which included it exactly as though it were your own contribution. Why is that?

Reply:
> @nayeemrmn I noticed that you declined to accept this suggested change

?

> but then later pushed a [commit](https://github.com/denoland/deno/pull/15339/commits/59738cad3520ed49afe8257a4b1c3379b11d97a4) which included it exactly as though it were your own contribution. Why is that?

You mean not using the commit suggestion feature? Sometimes I skip past that because I have to search and replace anyway in case there are test expectations to update. Pretty common, didn't realise it was so suspicious 😅 Sounds like you want attribution though so I'll reconstruct the branch

Reply:
Actually I've decided to leave the change out of this PR. This is a pre-existing error message which is unrelated to this change, please open your own PR to improve it. The changes here get squashed anyway.

Reply:
> This is a pre-existing error message which is unrelated to this change, please open your own PR to improve it.

Thanks — wilco.